_garden/love-letter-to-early-internet.md

```
@@ -285,58 +394,42 @@ The hit counter above is actually a working example from a site that still offer
-Others added decoration and description all over the page, growing quite tall and busy, like posters on the windows and door of a shop you might choose to enter:
+ Before search engines could essentially generate such a list on-demand for you for any category or query you could give them, this was an early way to find "the best" sites in a category! Their [88x31](#88x31--468x60) buttons would even render with each member site's actual rank on them, hit counter-style! 
+
+Many of the topsites I was in - and ran! - used the [aardvark topsites php](https://web.archive.org/web/20050409033331/http://www.aardvarkind.com/) script. By the 2000s, [PHP](https://en.wikipedia.org/wiki/PHP)-enabled web hosting was more available, and it was relatively easy for anyone to drop a folder of scripts into a directory and run their own topsite list! With administration panels to simplify member management and built-in analytics (and prestige), these were an attractive alternative to [webrings](#webrings).
 
 {% polaroid
-	early-web/splash-tall_saint-seiya-world.jpg
-	title="Saint Seiya World Splash Page from 2005"
-	link="https://web.archive.org/web/20051001082837/http://saintseiyaworld2.free.fr/"
-	archive="none"
+	"./early-web/topsites-ppntop50-2005.jpg"
+	title="Pokemon Palace Network Top 50 - in 2005"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)